core/src/main/scala/org/apache/spark/executor/ExecutorSource.scala

```diff
@@ -73,6 +75,29 @@ class ExecutorSource(threadPool: ThreadPoolExecutor, executorId: String) extends
     registerFileSystemStat(scheme, "write_ops", _.getWriteOps(), 0)
   }
 
+  /** Dropwizard metrics gauge measuring the executor's process CPU time.
```

Comment:
Nit: the comments should begin on the next line. But this is scaladoc syntax, and inside a code block, normally we just use `//` block comments.